.github/integration/tests/tests.sh

```diff
@@ -135,10 +138,10 @@ check_encypted_file "data_files_unenc/data_file.c4gh" "data_files_unenc/dir1/dat
 
 for k in data_files_unenc/data_file.c4gh data_files_unenc/dir1/data_file1.c4gh
 do
-    check_uploaded_file test/dummy/$uploadDir"/"$k $k
+    check_uploaded_file test/"$user"/$uploadDir"/"$k $k
 done
 
-out_ls=$(s3cmd -c testing/directS3 ls -r s3://test/dummy/testEncryptUpload/data_files_unenc/)
+out_ls=$(s3cmd -c testing/directS3 ls -r s3://test/"$user"/testEncryptUpload/data_files_unenc/)
 if ( ! echo "$out_ls" | grep -v -q "c4gh" ); then
     echo "No unencrypted files were uploaded during encrypt+upload"
 else
```

Comment:
No reason to use an intermediate variable here:
```suggestion
if ! s3cmd -c testing/directS3 ls -r "s3://test/$user/testEncryptUpload/data_files_unenc/" | grep -v -q 'c4gh'
then
    echo 'No unencrypted files were uploaded during encrypt+upload'
else
```